packages/datacommons-db/pyproject.toml

```diff
@@ -1,18 +1,12 @@
-[build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
-
 [project]
 name = "datacommons-db"
 dynamic = ["version"]
 description = 'Data Commons DB'
+license = "Apache-2.0"
 readme = "README.md"
-requires-python = ">=3.11"
-license = "MIT"
+requires-python = ">=3.12,<3.13"
```

Comment:
why is this >=3.12 but api is >=3.11? should they be the same and/or does it matter if they have the same lower bound?